cmd/ocitools/generate.go

```diff
@@ -83,19 +87,30 @@ var generateCommand = cli.Command{
 			return err
 		}
 
+		specgen.ExportOpts.Seccomp = context.Bool("seccomp-only")
+
 		if context.IsSet("output") {
-			output := context.String("output")
-			err = specgen.SaveToFile(output)
+			specgen.ExportOpts.Output = context.String("output")
 		} else {
-			err = specgen.Save(os.Stdout)
+			specgen.ExportOpts.Output = "STDOUT"
```

Comment:
On Fri, Jul 22, 2016 at 11:33:29AM -0700, Grant Seltzer Richman wrote:
> >  		} else {
> > -			err = specgen.Save(os.Stdout)
> > +			specgen.ExportOpts.Output = "STDOUT"
> 
> Not sure what you mean by keeping the current switch, can you clarify?

I meant keep the current:

  if context.IsSet("output") {
    output := context.String("output")
    err = specgen.SaveToFile(output)
  } else {
		err = specgen.Save(os.Stdout)
  }

which switches (if/else) in the caller instead of punting that choice
to exportConfig looking at specgen.ExportOpts.Output.